packages/webpack-cli/lib/utils/cli-flags.js

```diff
@@ -6,6 +8,43 @@ const ADVANCED_GROUP = 'advanced';
 const DISPLAY_GROUP = 'stats';
 const ZERO_CONFIG_GROUP = 'zero-config';
 
+// Extract all the flages being exported from core. A list of cli flags generated by core
+// can be found here https://github.com/webpack/webpack/blob/master/test/__snapshots__/Cli.test.js.snap
+let flagsFromCore = version.startsWith(5)
```

Comment:
I think we can simplify this, using `typeof webpack.cli !== 'undefined'`

Reply:
yes we can